melbourne-school-of-theology.csl

```diff
@@ -255,7 +294,7 @@
     <choose>
       <if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="any">
         <choose>
-          <if is-numeric="edition">
+          <if is-numeric="edition note">
```

Comment:
I'll have more tomorrow, but this is almost certainly not what you mean. What's the goal here of adding "note"?